Roys/app/src/main/java/com/dwg_karrier/roys/ContentSwipe.java

```diff
@@ -0,0 +1,117 @@
+package com.dwg_karrier.roys;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.Date;
+
+public class ContentSwipe extends AppCompatActivity {
+  private SectionsPagerAdapter mSectionsPagerAdapter;
+  private ViewPager mViewPager;
```

Comment:
Does `m` means the member variable?
In that case since we're not using the convention over the project I recommend to modify it into the one like others. Otherwise let men know what `m` stands for.